auth/README.md

```diff
@@ -1,38 +1,39 @@
 # Auth Package
 
-Auth package is used for request authentication and authorization for user to service
-and service to service communication. Auth is used for scenarios that require stricter
-validations, and usage restrictions. `auth.Client` provided by the package provides seamless
+Use `package auth` for request authentication and authorization for user-to-service
```

Comment:
We have package auth without backticks later in the doc. Let's choose one. It seems like we can just say "package auth" in plain text.